client/src/rpb/rpb.scss

```diff
@@ -93,35 +97,32 @@ li.select-list-item {
   }
 }
 
-
-.pagination-container {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  & > .pagination > li > span {
-    padding: 10px 16px;
-  } 
-}
-
-.row-opition-content-search > div > div > .input-lg {
-  height: 38px;
-  line-height: 0.9;
+.full-width {
```

Comment:
A small utility class like this should probably be in `common_css`. I think we've mostly stuck them in `site.scss` till now, but I feel like we should start collecting generic utility classes in something like `common_css/utility_classes.scss` or something.